packages/framework/src/Resources/views/Admin/Inline/Menu/menu.html.twig

```diff
@@ -1,6 +1,29 @@
 <ul class="js-toggle-menu navig navig--side">
 
     {% if isMultidomain() %}
+        <li class="js-toggle-menu-item navig__item">
+            <div class="box-dropdown">
+                <div
+                        class="box-dropdown__select box-dropdown__select--menu"
+                        title="{{ 'Switch locale'|trans }}"
+                >
+                    <i class="svg svg-flag"></i>
+                    <span class="box-dropdown__select__text">
+                        {{ localeFlag(app.user.selectedLocale) }}
+                    </span>
+                    <i class="box-dropdown__select__arrow box-dropdown__select__arrow--up"></i>
+                    <i class="box-dropdown__select__arrow box-dropdown__select__arrow--down"></i>
+                </div>
+                <ul class="js-toggle-menu-submenu box-dropdown__options">
+                    {% for locale in locales %}
+                        <li class="box-dropdown__options__item">
+                            <a href="{{ url('admin_localization_selectlocale', { locale: locale }) }}" class="box-dropdown__options__item__link" title="{{ languageName(locale) }}">{{ localeFlag(locale) }} {{ languageName(locale) }}</a>
```

Comment:
_nitpick_ 
We could display the currently selected locale without the link, but it's not really necessary

Reply:
I tried and failed as without the link, the styling of the item is different and I do not want to waste any more time on that :smile: